### PR TITLE
Do not install already available version

### PR DESCRIPTION
**Pull request by octplane**
Fix for #3

### CODE REVIEW

**Comment by octplane** [comment]
This PR fixes the issue, but the output of rbenv version is quite fragile to parse. Checking for `/Users/{{username}}/.rbenv/versions/{{version}}/bin/ruby` is probably much more elegant. Let me know what you think of it.

**Comment by rricard** [comment]
thanks ! Never had the occasion to not restart fresh when testing this part ...

File: roles/git/vars/main.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 # defaults for git: **must be overridden** !!!
 
-git_user_name: Bob Example
+git_user_name: Pierre Baillet
 
-git_user_mail: bob@example.com
+git_user_mail: oct@zoy.org
```
Comment (rricard) [inline roles/git/vars/main.yml:6]: should be changed in your xc-custom's all.yml, not here ...

**Comment by rricard** [comment]
I don't know, I still like your solution, we don't expect rbenv to suddenly change it's output ... Just strip your variable changes from this PR and I think we can merge that !

File: roles/git/vars/main.yml (hunk 1 above)
Comment (octplane) [inline roles/git/vars/main.yml:6]: I know, because it does not work in xc-custom (cf #5 ) :'(

Comment (octplane) [inline roles/git/vars/main.yml:6]: This commit should not be there. sorry.

Comment (rricard) [inline roles/git/vars/main.yml:6]: don't worry, let's just do another PR ! :)
